### ISSUE TITLE
Question: Node 실행 방법

### ISSUE THREAD

**Issue by HanSJin** [HanLabs/SSBarod-api#2]
## 개요 (Required)

clone 후 node 를 어떻게 실행하나요?

`$ node start` 입력 시 아래 오류가 출력됩니다ㅜㅠ
<img width="523" alt="image" src="https://user-images.githubusercontent.com/13335632/66285503-ebb41000-e907-11e9-8c92-9a8846cb076d.png">

`$ docker run` 입력하면 docker 를 아직 install 안해서인지 안되는데 도커도 따로 깔아야하나요?

## Links    

-

**Comment by rosd89** [comment]
현재 패키지 작업 진행전입니다.
작업 진행후 이슈 종료하겠습니다.

**Comment by HanSJin** [comment]
네~~ 이슈 종료한 후 여유 있다면, clone 부터 run 까지 필요한 스텝들(Ex install mysql, npm modules...?) 을 README에 정리 부탁쿠요~!
감사합니다
